src/components/Shades.vue

```diff
@@ -338,9 +339,10 @@ export default {
       let stepUp = this.groupOptions.stepUp
       let stepDown = this.groupOptions.stepDown
       let rgb = converter.hsl.rgb(hsl)
+      let baseShadeStop = this.baseShadeStop
 
       let shades = this.stops.map(stop => {
-        const distance = 5 - stop
+        const distance = baseShadeStop - stop
```

Comment:
I think this is the most important line to change since I started here and it seemed to take effect as soon as I messed with this number. Not sure how many of the other changes are necessary.